<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgVulkanAnalysis</class>
 <widget class="QDialog" name="dlgVulkanAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>772</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Vulkan Installation Analysis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="system">
        <attribute name="title">
         <string>System Info</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QToolBox" name="systemInfo">
           <property name="currentIndex">
            <number>5</number>
           </property>
           <widget class="QWidget" name="environment">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>708</width>
              <height>435</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Environment</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QTableWidget" name="envTable">
               <property name="showGrid">
                <bool>true</bool>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="hardware">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>708</width>
              <height>435</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Hardware</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QTableWidget" name="hardwareTable">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="execInfo">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>708</width>
              <height>435</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Executable Info</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QTableWidget" name="executableTable">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="vulkanDriverInfo">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>708</width>
              <height>435</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Vulkan Driver Info</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QTableWidget" name="vkDriverInfoTable">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="vulkanRuntimes">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>708</width>
              <height>435</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Vulkan Runtimes</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QTableWidget" name="vkRuntimesTable">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="lunarGSDKs">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>708</width>
              <height>435</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Vulkan SDKs</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QTableWidget" name="lunarGSDKTable">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="vulkanImplicitLayers">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>708</width>
              <height>435</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Vulkan Implicit Layers</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QTableWidget" name="implicitLayersTable">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="vulkanExplicitLayers">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>708</width>
              <height>435</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Vulkan Explicit Layers</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QTableWidget" name="explicitLayersTable">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="vulkanLayerSettings">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>708</width>
              <height>435</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Vulkan Layer Settings File</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QTableWidget" name="layerSettingsTable">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="api">
        <attribute name="title">
         <string>Vulkan API Calls</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QToolBox" name="toolBoxAPI">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="instance">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>708</width>
              <height>585</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Instance</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QTableWidget" name="instanceTable">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="physicalDevices">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>98</width>
              <height>89</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Physical Devices</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QTableWidget" name="physicalDevicesTable">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="logicalDevices">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>98</width>
              <height>89</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Logical Devices</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QTableWidget" name="logicalDevicesTable">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="cleanup">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>98</width>
              <height>89</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Cleanup</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QTableWidget" name="cleanupTable">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tests">
        <attribute name="title">
         <string>External Tests</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QTableWidget" name="externalTestsTable">
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
